### ISSUE TITLE
gnt-job list crashes badly if a job contains non-ASCII characters

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#483]
Originally reported of Google Code with ID 427.

```
The problem can be seen on Ganeti 2.6.2 and 2.7 at least.

<b>What steps will reproduce the problem?</b>
1. Try to create an instance containing non-ASCII characters in the name
  gnt-instance add -t drbd --disk 0:size=1G -I hail -o debian-image instance0.example.comµ
  (notice the µ at the end)
2. Execute: 
  gnt-job list

<b>What is the expected output? What do you see instead?</b>
The first command is expected to fail (it's unlikely for that name to be correct), but "gnt-job list" should produce the list of jobs as normal, indicating that the latest one has failed.

Instead, this is the output of the "gnt-job list" command:

Traceback (most recent call last):
  File "/usr/sbin/gnt-job", line 21, in <module>
    sys.exit(main.Main())
  File "/usr/lib/python2.6/dist-packages/ganeti/client/gnt_job.py", line 538, in Main
    return GenericMain(commands, aliases=aliases)
  File "/usr/lib/python2.6/dist-packages/ganeti/cli.py", line 2399, in GenericMain
    result = func(options, args)
  File "/usr/lib/python2.6/dist-packages/ganeti/client/gnt_job.py", line 111, in ListJobs
    qfilter=qfilter, isnumeric=True, cl=cl)
  File "/usr/lib/python2.6/dist-packages/ganeti/cli.py", line 3072, in GenericList
    verbose=verbose)
  File "/usr/lib/python2.6/dist-packages/ganeti/cli.py", line 2970, in FormatQueryResult
    table = FormatTable(result.data, columns, header, separator)
  File "/usr/lib/python2.6/dist-packages/ganeti/cli.py", line 3198, in FormatTable
    formatted = [col.format(value) for value, col in zip(row, columns)]
  File "/usr/lib/python2.6/dist-packages/ganeti/cli.py", line 2897, in __call__
    return self._fn(value)
  File "/usr/lib/python2.6/dist-packages/ganeti/client/gnt_job.py", line 65, in <lambda>
    "summary": (lambda value: ",".join(str(item) for item in value), False),
  File "/usr/lib/python2.6/dist-packages/ganeti/client/gnt_job.py", line 65, in <genexpr>
    "summary": (lambda value: ",".join(str(item) for item in value), False),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xb5' in position 49: ordinal not in range(128)
```
Originally added on 2013-04-16 13:18:37 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-16 13:18:56 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-05-02 13:58:07 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 04bcb621d9da8b49fe5c71a8fafb84e8fc4a59a9
Author: Helga Velroyen <helgav@google.com>
Date:   Fri May 3 10:28:35 2013 +0200

    gnt-job list: deal with non-ascii encoding in jobs
    
    This fixes issue 432. If a job summary contained a non-ascii character,
    gnt-job list crashed badly so far. This patch fixes the crash and makes
    gnt-job list display unicode characters properly.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>

```
Originally added on 2013-05-03 09:06:08 +0000 UTC.

Mentioned Issue with GoogleCode ID 432 has been migrated to GitHub with ID #488.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
